src/net.h

```diff
@@ -92,6 +93,7 @@ struct AddedNodeInfo
     CService resolvedAddress;
     bool fConnected;
     bool fInbound;
+    bool use_p2p_v2;
```

Comment:
Nit: Current convention is to prefix with `m_` so `m_use_p2p_v2`.

Reply:
Done in #24545.